src/mo_def/syntax.ml

```diff
@@ -22,6 +22,7 @@ type resolved_import =
 (* Identifiers *)
 
 type id = string Source.phrase
+type idm = (string, bool) Source.annotated_phrase
```

Comment:
Instead of being boolean blind we could have `type var_source = Mutable | Immutable` here. WDYT?

Reply:
Sure. there might even be a type you can reuse for that: Syntax.mut'.

Reply:
Yep, that's a great suggestion!

Reply:
Done: 53f1ec738930cfff761af7cc57599ab33f300e17